src/backend/marsha/core/serializers/video.py

```diff
@@ -582,6 +582,8 @@ def get_lti_url(self, obj):
             the LTI url to be used by LTI consumers
 
         """
-        return self.context["request"].build_absolute_uri(
-            reverse("video_lti_view", args=[obj.id])
-        )
+        request = self.context["request"]
+        url = request.build_absolute_uri(reverse("video_lti_view", args=[obj.id]))
```

Comment:
can you create a function building the URL depending the object targeted ? You are doing the same here and in the view.

Reply:
Yes, I was thinking about this too.
I'll create a src/backend/marsha/core/utils/url_utils.py